### DISCUSSION TITLE
Introducing Subper - The Free AI Subtitling Tool Powered by Whisper

### DISCUSSION THREAD

**Discussion by kevinwcwong** [openai/whisper#1172] [Show and tell]
Thanks to Whisper and Silero VAD.

We are thrilled to introduce Subper ([https://subtitlewhisper.com](https://subtitlewhisper.com)), a free AI subtitling tool, that makes it easy to generate and edit accurate video subtitles and audio transcription. Whether you're a content creator, educator, Subper is the perfect tool for you.

Best of all, Subper is free to use. We believe that accessibility should be available to everyone, and we're committed to making it easy for you to add subtitles to your videos. You don't need any programming skills or GPU to create subtitles.

If you find Subper useful, we invite you to upvote, share it on social media platforms and help us spread the word. We're always looking for ways to improve our tool, and your feedback is valuable to us.

Best regards,
The Subper Team

![dev-app subtitlewhisper com_projects_19fc545a-227b-472b-be99-f971ac9c421b_editor](https://user-images.githubusercontent.com/12548459/229066298-ca06ad2e-4f22-4789-9cb8-844849597e44.png)

**Comment by IlianP** [comment]
I would've tested it for my native language, but unfortunately it seems that it's not yet available in German.

**Reply by kevinwcwong** [reply]
thanks for the feedback, will add in soon!

**Reply by kevinwcwong** [reply]
added German support

**Comment by as1483** [comment]
You have an Indian flag for Indonesia lol

**Reply by kevinwcwong** [reply]
will be fixed soon, thanks for the feedback!

**Reply by kevinwcwong** [reply]
fixed, thanks!

**Comment by mayeaux** [comment]
Looks cool! I like the subtitle editor and ability to burn in the subtitles to the video, good luck on the project!

**Comment by atulbhatt-system32** [comment]
I would like to try it for my native language "Hindi" to transcribe my poetries but unfortunately it isn't there.

**Reply by kevinwcwong** [reply]
thanks for the feedback, will add within this week and update here when its ready!

**Reply by kevinwcwong** [reply]
added Hindi support

**Comment by Riemannb** [comment]
Will Russian be added?

**Reply by kevinwcwong** [reply]
thanks for the feedback, will add within this week and update here when its ready!

**Reply by kevinwcwong** [reply]
added Russian support

**Comment by Frosti7** [comment]
Would the "[Speaker diarization](https://github.com/openai/whisper/discussions/264)" function be added?

**Reply by kevinwcwong** [reply]
thanks for the feedback! may i learn more about your use case? since speaker diarization is usually not applicable to subtitling.
we also have a plan to introduce meeting transcription with speaker diarization, with meeting summary and action items generated.

**Comment by Frosti7** [comment]
Yes, this is for meeting transcription

**Comment by waterlemonstudios** [comment]
Is there a way to create subtitles for the videos over 15 mins in length?

**Reply by kevinwcwong** [reply]
its currently completely free, will add a payment tier enabling longer videos if users find it useful

**Reply by waterlemonstudios** [reply]
Thank you for the information. It's really useful if there was a payment tier:)

**Comment by kesavkolla** [comment]
Is there an API available so that we can connect via programmatically?

**Comment by howardpen9** [comment]
nice nice!!

**Comment by Lightingstormhkg** [comment]
Is there no way to delete our videos and projects to avoid a giant list?no button of delete

**Reply by kevinwcwong** [reply]
@Lightingstormhkg  Sorry for the inconvenience caused. According to the development schedule, delete function shall be available before Sep. In the meantime, feel free to contact us to delete the projects.

**Comment by zeitseeing** [comment]
Looks great! What library did you use to build the fancy subtitle editor?

**Reply by kevinwcwong** [reply]
@zeitseeing Thanks! We started building and enriching the editor from scratch ~3 years ago.

**Comment by kevinwcwong** [comment]
doing tests and will soon update to provide largev3 on https://subtitlewhisper.com/
will update again here when largev3 is deployed

**Comment by kevinwcwong** [comment]
an additional track (medium, with vad enabled) is now provided for the following languages - en, de, tr, es, fr, to reduce hallucination and improve accuracy.

**Comment by rodgermoore** [comment]
You can changed the title now:

~~Introducing Subper - The Free AI Subtitling Tool Powered by Whisper~~

# Introducing Subper - The Paid and really expensive AI Subtitling Tool Powered by Whisper

**Reply by kevinwcwong** [reply]
As the usage becoming much larger, we have to add subscription tier to support our going effort on optimization and cloud server cost. We still keep the free tier features to get the transcription and subtitle embedded for short videos.

**Reply by zeitseeing** [reply]
I just took a look at the prices and they do feel rather steep @rodgermoore . 

Just to compare, here are a few cheaper alternatives:

[audiogest.app](https://audiogest.app) => €4.00/hour (credit based)
[happyscribe.com](https://happyscribe.com) => €8.50/hour (with basic plan monthly)
[transcriptiontom.com](https://www.transcriptiontom.com)  => $1.20/hour (with monthly subscription)

(Full disclosure: I'm the founder of transcriptiontom.com ;-) )

**Reply by kevinwcwong** [reply]
kudos to everyone who brings more alternatives with different features for users!

Subper pricing includes
1. gpt enabled functions, e.g. punctuation and paragraphing (meeting transcription with speaker diarization and summary in upcoming release)
2. n-to-n language translation
3. subtitle embedding
4. 2 subtitle tracks generated w/w.o vad optimization for particular languages to make sure users can choose the best subtitle out of the box
5. subtitle editor

Will probably introduce cheaper pricing tier between free and Basic tier with less features in the future.

**Comment by Timwi** [comment]
What a rip-off. Stop calling it free when it’s not free, that’s fraud. You talk about “cloud server cost” but you would not have any such cost if you just let me download it and run it locally. Stop dangling carrots on a stick to hold people hostage and start thinking about how to actually contribute to the world without exploiting people.

**Reply by kevinwcwong** [reply]
@Timwi there's currently >80% of users use the free tier to help on their transcription and subtitle embedding.

**Reply by mikewesten** [reply]
Downloading the transcriptions simply as an .srt file requires a paid subscription. Ouch..

**Comment by lrq3000** [comment]
For info, there is [Subtitle Edit](https://github.com/SubtitleEdit/subtitleedit), which can use Whisper for offline transcription and then also [nllb-serve](https://github.com/thammegowda/nllb-serve) for offline translation to 200 languages. All these tools are free and open-source.
